### DISCUSSION TITLE
在“流量穿透”的网络环境下连接一会儿就断，断一会儿又自动连上，然后再断......请问这种环境下如何配置可以避免断流？

### DISCUSSION THREAD

**Discussion by wholenewx** [shadowsocks/shadowsocks-android#2779] [Q&A]
您好，作者

国内朋友的网络环境是这样的：https://i.imgur.com/BqJTgbF.png ，流量穿透，三大运营商都穿，访问大陆网络资源没有任何问题，但是通过shadowsocks访问外网非常痛苦， shadowsocks-android和电脑上的shadowsocks-Qt5一会儿就断，断了过一会又自动连上，然后再断.....朋友在家里上外网痛苦，跟我语音连线痛苦，我放他家里的一台用来翻回国的小主机连接也痛苦。

现在可以确定的是shadowsocks断流的时候直接访问的bilibili播放正常，从国内网站的下载也没有中断。

国外的节点一直是安装的clowwindy的ss版本，配合kcptun，已经这样使用了好几年，翻回国的小主机也是安装的clowwindy的服务端配合zerotier使用，朋友搬家到现地址后发现都不好用了。

kcptun的配置很简单：mode=fast2;remoteaddr=xxx.xxx.xx.xx:yyyyy 
shadowsocks的配置也很简单：ssserver -p xxxxx -k xxxxxxx -m aes-256-cfb

请问：对付这种流量穿透的宽带，有没有什么特别的配置方式可以解决断流的问题？或者您有没有什么建议？谢谢！

**Comment by sophauer** [comment]
换机场可破

**Comment by wholenewx** [comment]
> 换机场可破

我们有几个VPS，不想换机场，还是倾向于从技术上解决问题
